src/helpers/getSectionColors.ts

```diff
@@ -0,0 +1,20 @@
+/* Retrieves section color values.
+ *
+ * Requires a section type as defined by sectionDataMap. Accepts an optional color
+ * preference ("primary" or "secondary").
+ *
+ * @returns A string if one color is requested or an object containing both (default).
```

Comment:
Can be a little more specific here:

@returns A string if one color is requested (by passing a `colorVal`) or an object containing both (default).

Reply:
OK. Rewritten.